dds_cli/user.py

```diff
@@ -312,40 +309,30 @@ def token_report(self, token_metadata=None):
         LOG.info(f"[{markup_color}]{sign}  {message} {sign} [/{markup_color}]")
         LOG.info(f"[{markup_color}]Token age: {readable_timedelta(age)}[/{markup_color}]")
 
-        if lifetime.total_seconds() < 0:
+        if age > dds_cli.TOKEN_MAX_AGE:
+            LOG.info(f"[{markup_color}]Token expired: {expiration_time}[/{markup_color}]")
+        else:
+            lifetime = age - dds_cli.TOKEN_MAX_AGE
             LOG.info(
                 f"[{markup_color}]Token expires: {expiration_time} (in {readable_timedelta(lifetime)})[/{markup_color}]"
             )
-        else:
-            LOG.info(f"[{markup_color}]Token expired: {expiration_time}[/{markup_color}]")
-
-        if consignee:
```

Comment:
I am not happy that you dropped the whole consignee functionality. Even though I certainly have not the last say on this, I think it was very welcome from a UX perspective. 

Also mind that [Anands PR](https://github.com/ScilifelabDataCentre/dds_cli/pull/244) relies on this funcitonality.

Reply:
Already replied to this